mindsdb/integrations/handlers/openai_handler/openai_handler.py

```diff
@@ -693,8 +691,6 @@ def finetune(
         api_key = get_api_key(self.name, args, self.engine_storage)
 
         using_args = args.pop('using') if 'using' in args else {}
-        prompt_col = using_args.get('prompt_column', 'prompt')
```

Comment:
Curious as to why these were removed, are they not used?

Reply:
Yes, prompt_col and completion_col were not referenced anywhere